test/WebSites/ModelBindingWebSite/Controllers/VehicleController.cs

```diff
@@ -43,6 +59,74 @@ public IActionResult UpdateDealerVehicle(VehicleWithDealerViewModel model)
             return PartialView("UpdateSuccessful", model);
         }
 
+        [HttpGet("/vehicles/{id:int}")]
+        public IActionResult Details(int id)
+        {
+            VehicleViewModel vehicle;
+            if (!_vehicles.TryGetValue(id, out vehicle))
```

Comment:
don't we want to be somewhat realistic in functional tests?

then again I've skipped the dictionary and used a single instance in another functional test.  I'll do the same here.